### PR TITLE
releng: Image promotion for Kubernetes v1.18.10 / v1.18.11-rc.0

### PR DESCRIPTION
**Pull request by jimangel**
Cloudbuild stage: ([logs](https://console.cloud.google.com/cloud-build/builds/afedad50-59c3-4098-b715-875236bc712a?project=648026197307))

Commands:
```
cip-mm --base_dir=$GOPATH/src/github.com/jimangel/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.18.10

cip-mm --base_dir=$GOPATH/src/github.com/jimangel/k8s.io/k8s.gcr.io --staging_repo=gcr.io/k8s-staging-kubernetes --filter_tag=v1.18.11-rc.0

# removed the part of the mock images.
```

@kubernetes/release-engineering
/assign @justaugustus @saschagrunert @hasheddan @cpanato
/hold

### CODE REVIEW

File: k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml [hunk 1]
```diff
@@ -1108,6 +1180,78 @@
     "sha256:e15927e26974af0e066018b056c5f51e514347950e164e4e5be822ed723ee7e0": ["v1.17.10"]
     "sha256:f5de4ad913ffd66ceee261f40fecaec852342168596c9e6d2d8c6fe9f1cd7359": ["v1.18.9"]
     "sha256:f86066539c4a8fb82715e049ef13c0161dbc3b534881b2315476b254b0556733": ["v1.19.1"]
+- name: mock/conformance
```
Comment (hasheddan) [inline k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:1183]: Hmm looks like these didn't get removed :thinking:

Comment (cpanato) [inline k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:1183]: please remove all this mock/*

Comment (cpanato) [reply k8s.gcr.io/images/k8s-staging-kubernetes/images.yaml:1183]: https://github.com/kubernetes/k8s.io/pull/1337/files#diff-36044de293c1633f93272919ab4ce0c69c57831ec1d4517f08ff84fcf694d87eR1183-R1254

**Review by cpanato — APPROVED** [review]
/lgtm

**Pull request merged**